directory/directory_test.go

```diff
@@ -47,11 +51,10 @@ func TestGetPutManifest(t *testing.T) {
 	assert.Equal(t, man, m)
 	assert.Equal(t, "", mt)
 
-	// Non-default instances are not supported
-	md, err := manifest.Digest(man)
-	require.NoError(t, err)
 	_, _, err = src.GetManifest(&md)
```

Comment:
… to make sure the `GetManifest`/`PutManifest` pairs read/write the correct ones, or, in this case, that the test does not ignore the return value of `GetManifest(&md)` by using `_, _, err`.

Reply:
Agreed, fixed.